src/controller/userController.js

```diff
@@ -1,11 +1,79 @@
-getUser = async (req, res, next) => {};
-addUser = async (req, res, next) => {};
-updateUser = async (req, res, next) => {};
-deleteUser = async (req, res, next) => {};
+const User = require('../models/User');
+
+// for testing
+const getUsers = async (req, res, next) => {
+  try {
+    const allUsers = await User.find();
+    res.json(allUsers);
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
I actually wanted to ask about if we are using this function or not. I think what I said apply for both

Reply:
I understand, you want to use it, but main it is branch that it shouldn't be used for that. Now we would have a 10 lines function that you could easily make and we are not using, you can create a branch for that and leave there if you don't want to delete it. We should only merge to main production-code, not testing code.

Reply:
Always think the code has to follow some logic, at least the code in `main`, right now the rest of the team is seeing a function that is declared but never used. So if that is in main I will always wonder what we are not using it. If something is not being used, shouldn't be in main.